### PR TITLE
{lib}[GCCcore/10.3.0] muparserx v4.0.8

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)
Not sure about what capitalization to use. Author uses all kinds of variants.

### CODE REVIEW

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
vera-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/ab10499744f82fde5165b4ef44af28c5 for a full test report.

**Comment by Micket** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/m/muparserx/muparserx-4.0.8-GCCcore-10.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+easyblock = 'CMakeMake'
+
+name = 'muparserx'
+version = '4.0.8'
+
+homepage = 'https://beltoforion.de/en/muparserx/'
+description = 'Very fast, header-only/compiled, C++ logging library.'
```
Comment (branfosj) [inline easybuild/easyconfigs/m/muparserx/muparserx-4.0.8-GCCcore-10.3.0.eb:7]:
```suggestion
description = 'A C++ Library for Parsing Expressions with Strings, Complex Numbers, Vectors, Matrices and more.'
```

**Review by branfosj — APPROVED** [review]
lgtm

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/0166fc653b77029afea5bd23420f107e for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/01f44496c02a6a177a0dcf1f8c32a369 for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @Micket!

**Pull request merged**